components/chart/types/column.md

```diff
@@ -59,6 +59,17 @@ Column series
 
 ## Column Chart Specific Appearance Settings
 
+### Markers
+
+Each data item is denoted with a marker (label). You can control and customize them through the `< ChartCategoryAxisLabels />` and its children tags.
```

Comment:
The wording should be something like "Each item can have a label", because the markers are something different in the context of line-type series

Reply:
Also, there is an extra space after the `<` bracket

Reply:
The heading should be something like `Series Labels`. not markers